packages/neuron-ui/src/widgets/Pagination/pagination.module.scss

```diff
@@ -1,7 +1,7 @@
-$primary-color: #1b73b3;
-$secondary-color: #717171;
+$primary-color: #666;
```

Comment:
Should consider `light` and `dark` mode, use `var(variable)`, the variable can be found in `packages/neuron-ui/src/styles/theme.scss`